Services/CustomersService.cs

```diff
@@ -19,14 +19,12 @@ public Customer Authenticate(CustomerLogin customerLoginDto)
             return currentUser;
         }
 
-        protected override Customer ModelToEntity(CustomerLogin dto, int id = 0)
+        protected override Customer ModelToEntity(CustomerCreate dto, int id = 0)
```

Comment:
Do these changes not break `/Auth/Login` endpoint?

Reply:
nope, login method does not use ModelToEntity, since it does not actually create or modify any entities